govuk_sidekiq.gemspec

```diff
@@ -19,15 +19,16 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.2"
 
-  spec.add_dependency "sidekiq", "~> 4.2.8"
+  spec.add_dependency "sidekiq", ">= 4.2", "< 6"
```

Comment:
Maybe it's good to keep this specific? It'd be great if we can get all apps running the same Sidekiq version.

Reply:
@tijmenb I'm not sure about that - it wondering if that creates a maintenance burden on this gem to keep sidekiq up to date that would be easier managed by the much more frequently checked apps.

E.g if we assume we have dependabot in the mix, a dependency like this will mean each of the apps get a PR to update whenever a new version of sidekiq is released (below version 6) so that should keep them up to date. Whereas if they're tied to a sidekiq through this then we need to go through the release process of this gem each time there's a new sidekiq version (also I couldn't determine whether dependabot supports gems)

Reply:
Interesting! My thinking was that this gem would be used to "vet" dependency upgrades. In an ideal world, any user of `govuk_sidekiq` can update to the latest version and be confident that it's very likely to work with GOV.UK's setup. The idea being that any upgrade trouble is solved centrally, instead of repeatedly in each application.

I've turned on Dependabot for this repo - https://github.com/alphagov/govuk_sidekiq/pulls

Reply:
Ok cool - well I'll give that a spin and leave a note in the commit message that if we find it's neglected or problematic (i.e. an app needs a sidekiq extension that isn't compatible with this) we can switch that round.